File: ARRAY AND HASHMAP/PlusOne.py
class Solution(object):
    def PlusOne(self, digits):
        for i in range(len(digits)-1, -1, -1):
            digits[i]+=1
            digits[i] %= 10
            if(digits[i]!=0):
                return digits
        return [1] + digits
s1 = Solution()
digits = [1,2,3,4,5]
result = s1.PlusOne(digits)
print(result)

File: ARRAY AND HASHMAP/PlusOne.cpp
#include <iostream>
#include <vector>
using namespace std;
class Solution{
    public:
        int PlusOne(vector<int> digits){
            for(int i = digits.size() - 1; i >= 0; i++){
                digits[i]++;
                digits[i]%=10;
                if(digits[i] == 0){
                    return digits;
                }
            }
            return 
        }
};